### PR TITLE
Add [GitHubLastCommit] by committer badge

### LINKED ISSUE #6236: Badge request: GitHub last-commit variant based on "committer" timestamp
Are you experiencing an issue with...

- [x] [shields.io](https://shields.io/#/)
- [ ] My own instance
- [ ] [badge-maker NPM package](https://www.npmjs.com/package/badge-maker)

:beetle: **Description**

Some commits can have a different "commit created" date and "committed date". This is especially seen when cherry-picking. In this case, the last commit badge shows the date of commit creation rather than the "committed date". 
image

I don't know if this is intended behaviour - hence why I am opening this as a bug report. If this is indeed how it is expected to work, please consider this as a feature request to add a badge of the latter

:link: https://img.shields.io/github/last-commit/yt-dlp/yt-dlp/master

Last Commit 
Since the badge will change as soon as I push a new commit, I am also attaching a screenshot of how it looks like as of https://github.com/yt-dlp/yt-dlp/commit/097b056c5a4f0374595798db0c663b9282acdf01
image

### PR DESCRIPTION
**Pull request by PaulaBarszcz**
closes #6236

I added  the `display_timestamp` queryParam, with options `author` and `committer`, defaulting to `author`.

```js

const queryParamSchema = Joi.object({
  display_timestamp: Joi.string()
    .valid('author', 'committer')
    .default('author'),
}).required()

...

  async handle({ user, repo, branch }, queryParams) {
    const body = await this.fetch({ user, repo, branch })

    return this.constructor.render({
      commitDate: body[0].commit[queryParams.display_timestamp].date,
    })
  }
```

I checked a couple of repos (for example https://api.github.com/repos/zero-to-mastery/start-here-guidelines/commits) but I couldn't find an example with a different date for `author` and `committer`.

http://localhost:8080/github/last-commit/google/skia/infra/config?display_timestamp=committer

image

### CODE REVIEW

File: services/github/github-last-commit.service.js [hunk 1]
```diff
@@ -15,14 +15,26 @@ const schema = Joi.array()
...
+        committer: Joi.object({
+          date: Joi.string().required(),
+        }).required(),
       }).required(),
     }).required()
   )
   .required()
 
+const queryParamSchema = Joi.object({
+  by_committer: Joi.equal(''),
```
Comment (chris48s) [inline services/github/github-last-commit.service.js:27]: Similar to https://github.com/badges/shields/pull/8543#discussion_r998655480 I'd make this `display_timestamp` with options `author` and `committer` defaulting to `author`

Comment (PaulaBarszcz) [reply services/github/github-last-commit.service.js:27]: Good point, thanks. Updated

**Comment by PaulaBarszcz** [comment]
Hi @chris48s 
For some reason integration tests failed here:
image
https://github.com/badges/shields/actions/runs/3290599111/jobs/5423657553

**Review by chris48s — APPROVED** [review]

**Comment by pukkandan** [comment]
Thanks

**Pull request merged**